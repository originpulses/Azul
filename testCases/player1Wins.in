2
saveGame.txt
turn 1 U 1
turn 2 U 5
turn 4 O 6
turn 3 U 5
turn 0 B 2
turn 0 R 2
turn 5 L 5
turn 0 Y 4
turn 0 U 3
turn 0 R 6
6
6
6
6
6
4
