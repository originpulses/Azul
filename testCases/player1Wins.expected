Menu
----
1. New Game
2. Load Game
3. Credits (Show student information)
4. Quit

> Enter the filename from which you want to load a game
> Azul game successfully loaded
< game play continues from here > 

			 *INSTRUCTIONS*

			Input formats: <type> <factory> <colour> <storage row>

			1. First you will type the command name e.g. turn, save or -1.
			   only these 3 are valid command names

			2. Then with a SPACE you will write the number of factory from which you
			   want to pick tiles. (This MUST be an integer (range 0 - 5) )

			3. Then with a SPACE you will write the colour of tile you want to pick
			   from that specified factory. (Valid colours: R, Y, B, L, U) all other inputs are invalid.

			4. Then at last, with a SPACE you will write the numebr of storage row.
			   (This MUST be an integer (range 1 - 5) )

			5. At any moment a player can enter -1 to exit the game and terminate 
			   the program. (CAUTION: All unsaved progress will be lost)

	=== Start Round ===
TURN FOR PLAYER:PLAYER_2
Factories:
0: F 
1: B Y Y R 
2: B Y R U 
3: L L Y B 
4: R U R B 
5: L Y B B 

Mosaic for PLAYER_2
1:         .  || B Y R . L 
2:       . .  || . B . . . 
3:     . . .  || . L . . . 
4:   . . . .  || . U . . . 
5: . . . . .  || . . . . . 
Broken: 

> Turn successful.

TURN FOR PLAYER:PLAYER_1
Factories:
0: F L B B 
1: B Y Y R 
2: B Y R U 
3: L L Y B 
4: R U R B 
5: 

Mosaic for PLAYER_1
1:         .  || B Y R U . 
2:       . .  || L . . . . 
3:     . . .  || U . . . . 
4:   . . . .  || R . . . . 
5: Y Y . . .  || . . . . . 
Broken: 

> PLAYER_1 got the first player tile.
Turn successful.

TURN FOR PLAYER:PLAYER_2
Factories:
0: B B 
1: B Y Y R 
2: B Y R U 
3: L L Y B 
4: R U R B 
5: 

Mosaic for PLAYER_2
1:         .  || B Y R . L 
2:       . .  || . B . . . 
3:     . . .  || . L . . . 
4:   . . . .  || . U . . . 
5: Y . . . .  || . . . . . 
Broken: 

> Turn successful.

TURN FOR PLAYER:PLAYER_1
Factories:
0: B B U B 
1: B Y Y R 
2: B Y R U 
3: L L Y B 
4: 
5: 

Mosaic for PLAYER_1
1:         L  || B Y R U . 
2:       . .  || L . . . . 
3:     . . .  || U . . . . 
4:   . . . .  || R . . . . 
5: Y Y . . .  || . . . . . 
Broken: F 

> Turn successful.

TURN FOR PLAYER:PLAYER_2
Factories:
0: B B U B B R 
1: 
2: B Y R U 
3: L L Y B 
4: 
5: 

Mosaic for PLAYER_2
1:         .  || B Y R . L 
2:       . .  || . B . . . 
3:     . . .  || . L . . . 
4:   R R . .  || . U . . . 
5: Y . . . .  || . . . . . 
Broken: 

> Turn successful.

TURN FOR PLAYER:PLAYER_1
Factories:
0: U R 
1: 
2: B Y R U 
3: L L Y B 
4: 
5: 

Mosaic for PLAYER_1
1:         L  || B Y R U . 
2:       . .  || L . . . . 
3:     . . .  || U . . . . 
4:   . . . .  || R . . . . 
5: Y Y Y Y .  || . . . . . 
Broken: F 

> Turn successful.

TURN FOR PLAYER:PLAYER_2
Factories:
0: U R B R U 
1: 
2: 
3: L L Y B 
4: 
5: 

Mosaic for PLAYER_2
1:         .  || B Y R . L 
2:       . .  || . B . . . 
3:     B B B  || . L . . . 
4:   R R . .  || . U . . . 
5: Y . . . .  || . . . . . 
Broken: B 

> Turn successful.

TURN FOR PLAYER:PLAYER_1
Factories:
0: U B U 
1: 
2: 
3: L L Y B 
4: 
5: 

Mosaic for PLAYER_1
1:         L  || B Y R U . 
2:       . .  || L . . . . 
3:     . . .  || U . . . . 
4:   . . . .  || R . . . . 
5: Y Y Y Y Y  || . . . . . 
Broken: F 

> Turn successful.

TURN FOR PLAYER:PLAYER_2
Factories:
0: B 
1: 
2: 
3: L L Y B 
4: 
5: 

Mosaic for PLAYER_2
1:         .  || B Y R . L 
2:       . .  || . B . . . 
3:     B B B  || . L . . . 
4:   R R R R  || . U . . . 
5: Y . . . .  || . . . . . 
Broken: B 

> Turn successful.

TURN FOR PLAYER:PLAYER_1
Factories:
0: B Y B 
1: 
2: 
3: 
4: 
5: 

Mosaic for PLAYER_1
1:         L  || B Y R U . 
2:       U U  || L . . . . 
3:     . . .  || U . . . . 
4:   . . . .  || R . . . . 
5: Y Y Y Y Y  || . . . . . 
Broken: F 

> Turn successful.

TURN FOR PLAYER:PLAYER_2
Factories:
0: Y 
1: 
2: 
3: 
4: 
5: 

Mosaic for PLAYER_2
1:         .  || B Y R . L 
2:       L L  || . B . . . 
3:     B B B  || . L . . . 
4:   R R R R  || . U . . . 
5: Y . . . .  || . . . . . 
Broken: B 

> Turn successful.

	=== END OF ROUND ===

***** Score board of player PLAYER_1

Total points deducted for broken tiles: -1.
Score of this round: 11.
Total score of after this round: 30.

***** Score board of player PLAYER_2

Total points deducted for broken tiles: -1.
Score of this round: 7.
Total score of after this round: 23.

=== GAME OVER ===


Player PLAYER_1 Wins!

Score of PLAYER_1 = 49
Score of PLAYER_2 = 33

 ***** Moving towards the main menu *****
Menu
----
1. New Game
2. Load Game
3. Credits (Show student information)
4. Quit

> 

Goodbye
